docs/getting-started/install-zowe-sdks.md

```diff
@@ -0,0 +1,98 @@
+# Zowe SDKs installation
+
+Leverage the Zowe Client Software Development Kits (SDKs) to build client applications and scripts that interface with the mainframe.
+
+The SDKs include programmatic APIs, each of which performs a particular mainframe task. For example, one API package provides the ability to upload and download z/OS data sets. You can leverage that package to rapidly build a client application that interacts with data sets.
+
+The following SDKs are available.
+- Zowe Client Java SDK
+- Zowe Client Kotlin SDK
+- Zowe Client Node.js SDK
+- Zowe Client Python SDK *technical preview*
+
+## SDK documentation
+
+For detailed SDK documentation, see the following:
+- [Zowe Client Java SDK](https://github.com/Zowe-Java-SDK)
+- [Zowe Client Node.js SDK](https://docs.zowe.org/stable/typedoc/index.html)
+- [Zowe Client Python SDK](https://zowe-client-python-sdk.readthedocs.io/en/latest/) *technical preview*
+
+## Installation and software requirements
+
+### Java
```

Comment:
It might be good to say "Java SDK", "Kotlin SDK", etc. Because otherwise it kind of sounds like JRE 17 is a requirement for all SDKs.